fitnessllm_dataplatform/utils/logging_utils.py

```
@@ -41,0 +1,131 @@
+
+        # Add any additional fields
+        log_data.update(kwargs)
+
+        return log_data
+
+    def info(
+        self,
+        message: str,
+        data_source: Optional[str] = None,
+        user_id: Optional[str] = None,
+        **kwargs
```

Ruff (0.8.2)

Missing type annotation for `**kwargs`

(ANN003)